app/javascript/components/ItemCard.jsx

```diff
@@ -10,53 +10,96 @@ export default (props) => {
 	const [onHover, setOnHover] = useState(false);
 	const [errorMessage, setErrorMessage] = useState(null);
 	const [cookies, _setCookie, _removeCookie] = useCookies(["ddough-auth"]);
+	// Unnecessary.
```

Comment:
?

Reply:
An unnecessary comment about something that I thought was unnecessary.